src/nvim/option_vars.h

```diff
@@ -530,6 +530,8 @@ EXTERN char *p_mef;             ///< 'makeef'
 EXTERN char *p_mp;              ///< 'makeprg'
 EXTERN char *p_mps;             ///< 'matchpairs'
 EXTERN OptInt p_mat;            ///< 'matchtime'
+EXTERN OptInt p_mco;            ///< 'maxcombine'
+#define MAX_MCO  6  // fixed value for 'maxcombine'
```

Comment:
why is this defined in 2 different places?

Reply:
> why is this defined in 2 different places?

That was an error on my part. Fixed.